### PR TITLE
add app-specific favicon for better recognition among tabs

### PR DESCRIPTION
**Pull request by jancborchardt**
Check it @LukasReschke @Ivansss @nickvergessen @MorrisJobke

### CODE REVIEW

File: img/favicon-mask.svg [hunk 1]
```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg xmlns="http://www.w3.org/2000/svg" xml:space="preserve" height="16" viewBox="0 0 16 15.999999" width="16" version="1.1" y="0px" x="0px" enable-background="new 0 0 595.275 311.111"><path d="m2.5 0c-1.385 0-2.5 1.115-2.5 2.5v11c0 1.385 1.115 2.5 2.5 2.5h11c1.385 0 2.5-1.115 2.5-2.5v-11c0-1.385-1.115-2.5-2.5-2.5h-11zm0.5 3h6c1.108 0 2 0.892 2 2v1.334l4-3.334v10l-4-3.334v1.334c0 1.108-0.892 2-2 2h-6c-1.108 0-2-0.892-2-2v-6c0-1.108 0.892-2 2-2z"/></svg>
```
Comment (nickvergessen) [inline img/favicon-mask.svg]: fix the viewbox please to 16 instead of 15.99999999

File: img/favicon-touch.svg [hunk 2]
```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg xmlns="http://www.w3.org/2000/svg" xml:space="preserve" height="128" viewBox="0 0 128 127.99999" width="128" version="1.1" y="0px" x="0px" enable-background="new 0 0 595.275 311.111"><rect rx="20" ry="20" height="128" width="128" y="-.0000015" x="0" fill="#0082c9"/><path fill="#fff" d="m24 24c-8.864 0-16 7.136-16 16v48c0 8.864 7.136 16 16 16h48c8.864 0 16-7.136 16-16v-10.672l32 26.672v-80l-32 26.672v-10.672c0-8.864-7.136-16-16-16z"/></svg>
```
Comment (nickvergessen) [inline img/favicon-touch.svg]: same here

File: img/favicon.svg [hunk 3]
```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg xmlns="http://www.w3.org/2000/svg" xml:space="preserve" height="32" viewBox="0 0 32 31.999997" width="32" version="1.1" y="0px" x="0px" enable-background="new 0 0 595.275 311.111"><rect rx="5" ry="5" height="32" width="32" y="-.0000052588" x="0" fill="#0082c9"/><path fill="#fff" d="m6 6c-2.216 0-4 1.784-4 4v12c0 2.216 1.784 4 4 4h12c2.216 0 4-1.784 4-4v-2.668l8 6.668v-20l-8 6.668v-2.668c0-2.216-1.784-4-4-4z"/></svg>
```
Comment (nickvergessen) [inline img/favicon.svg]: and here

File: img/favicon-mask.svg (hunk 1 above)
Comment (jancborchardt) [inline img/favicon-mask.svg]: Woah why did that happen :D

**Comment by jancborchardt** [comment]
Fixed, will also fix in the other apps where this was always copied over from.

**Comment by nickvergessen** [comment]
Btw this is not themed anymore then, right? 😞

**Comment by jancborchardt** [comment]
Good question actually, I thought maybe https://github.com/nextcloud/server/pull/840 by @juliushaertl fixes this? :) Because we have custom favicons in Mail, Calendar, Contacts and Notes since some time already.

**Pull request merged**